staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1220,6 +1220,14 @@ type KubeVirtSpec struct {
 
 	// The ImagePullPolicy to use.
 	ImagePullPolicy k8sv1.PullPolicy `json:"imagePullPolicy,omitempty" valid:"required"`
+
+	// The namespace Prometheus is deployed in
+	// Defaults to openshift-monitor
+	monitorNamespace string `json:"monitorNamespace,omitempty"`
```

Comment:
field name needs to be uppercase, else it won't be (de-)serialized. Actually this is the reason why a json tag is always added AFAIK: field name needs to be uppercase, best practice for json keys is lowercase. Otherwise json key would default to field (uppercase) field name.